specification/eventgrid/data-plane/Microsoft.Storage/stable/2018-01-01/Storage.json

```diff
@@ -325,6 +325,34 @@
           "description": "The type of blob.",
           "type": "string"
         },
+        "accessTier": {
+          "description": "The current tier of the blob.",
+          "type": "string",
+          "enum": [
```

Comment:
Rather than repeat the enum inline, you can add a definition and then reference it. You can look at the GeneratedSystemEvents changes to see how that is done. Also, I think we should name the enum "StorageBlobAccessTier".

Reply:
Changed the enum definition and referenced it in the 2 places.